native/packages/react-native-bpk-component-image/src/BpkImage-test.common.js

```diff
@@ -0,0 +1,124 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import renderer from 'react-test-renderer';
+
+import BpkImage from './BpkImage';
+
+const commonTests = () => {
+  describe('BpkImage', () => {
+    it('should render correctly', () => {
+      const tree = renderer.create(<BpkImage />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with custom style', () => {
+      const styles = StyleSheet.create({
+        custom: {
+          flex: 1,
+        },
+      });
+
+      const tree = renderer
+        .create(
+          <BpkImage
+            style={styles.custom}
+            alt="test"
+            source={{
+              uri:
+                'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
+            }}
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with arbitrary props', () => {
+      const tree = renderer
+        .create(
+          <BpkImage
+            testID="123" // <- Arbitrary prop.
+            alt="test"
+            source={{
+              uri:
+                'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
+            }}
+          />,
+        )
+        .toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly when not loaded', () => {
+      const tree = renderer
+        .create(
+          <BpkImage
+            loaded={false}
+            alt="test"
+            source={{
+              uri:
+                'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
+            }}
+          />,
+        )
+        .toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly without border radius', () => {
+      const tree = renderer
+        .create(
+          <BpkImage
+            borderRadius={false}
+            alt="test"
+            source={{
+              uri:
+                'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
+            }}
+          />,
+        )
+        .toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly when unloaded and out of view', () => {
+      const tree = renderer
+        .create(
+          <BpkImage
+            inView={false}
+            loaded={false}
+            alt="test"
+            source={{
+              uri:
+                'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
```

Comment:
Because you're using the same `alt` and `source` props for every test, you could make them a bit DRYer and reduce visual noise by putting these into a variable that you can reuse in each test:

```
const requiredProps = {
  alt: 'test',
  source: {
    uri: 'https....",
  },
};

...

<BpkImage {...requiredProps} inView={false} />
```